aircraft/ball/ball.xml

```diff
@@ -55,7 +55,7 @@
         <ixy unit="SLUG*FT2"> -0 </ixy>
         <ixz unit="SLUG*FT2"> -0 </ixz>
         <iyz unit="SLUG*FT2"> -0 </iyz>
-        <emptywt unit="LBS"> 40 </emptywt>
+        <emptywt unit="LBS"> 20000 </emptywt>
```

Comment:
Yep, I actually thought we had a specific test using the ball, so sort of expected one of the tests to fail, but they all passed.

Reply:
Well, the test passes in the meaning that it does not crash (numerically speaking :wink:). Actually the scripts ends gracefully because it is protected by the *Terminate* event:
https://github.com/JSBSim-Team/jsbsim/blob/c9b12c4794046ae1406e1a66941937a64417bfd5/scripts/ball_chute.xml#L45-L54

However, if the empty weight is increased from 40 lbs to 20,000 lbs then the ball reaches the ground in 37 sec (it currently hits the ground in a bit less than 108 sec). Strictly speaking there is nothing wrong with that except that the parachute may look a bit undersized for a 20,000 lbs vehicle :smile:

If we want to keep the PR as it currently is, I'm perfectly fine with that. I was just raising a flag so that the decision is made purposely.

Reply:
I thought there was a test that compared the data output from a previous reference run and so given the change in mass the output wouldn't match.

Reply:
> I thought there was a test that compared the data output from a previous reference run and so given the change in mass the output wouldn't match.

Actually there is only one test that make that kind of regression test: [tests/RunCheckCases.py](https://github.com/JSBSim-Team/jsbsim/blob/master/tests/RunCheckCases.py).

Reply:
Ah, yep, just found that a little while ago. However when running the `ball_orbit.xml` script I noticed that the CSV file produced is empty, it only contains the header line. I ran it twice with a varying mass for the ball to confirm that there was no diff between the two CSV files. And there isn't, because they're both empty!

Taking a closer look.  In the aircraft file, `ball.xml` we have a 1Hz output set.

```xml
    <output name="BallOut.csv" type="CSV" rate="1">
        <rates> ON </rates>
        <velocities> ON </velocities>
        <forces> ON </forces>
        <moments> ON </moments>
        <position> ON </position>
        <massprops> ON </massprops>
        <property> position/vrp-gc-latitude_deg </property>
        <property> position/vrp-longitude_deg </property>
        <property> position/vrp-radius-ft </property>
    </output>
```

However, the script `ball_orbit.xml` seems to disable CSV output via this property.

```xml
<property value="0"> simulation/output[0]/log_rate_hz </property>
```

Reply:
> However, the script `ball_orbit.xml` seems to disable CSV output via this property.

That's most likely a remnant of a debug session according to the commit eed5f45726a3a8938f1230c938449919aba19cca that disabled the CVS output.

Reply:
So, when I make the suggested changes, how do I get those up to the repo so they can be incorporated, when the time comes? 

- Commit
- Push

??